### PR TITLE
glTF 1.0.1 Adds support for normalized vertex arrays and a test

### PR DESCRIPTION
**Pull request by lasalvavida**
Part of #4009. Test model is quantized box with a high precision decode matrix via the glTF 1.0.1 normalized property.

### CODE REVIEW

**Comment by lasalvavida** [comment]
@pjcozzi, could you look at this if you get a chance so we can start getting this chain of pull requests merged?

**Comment by pjcozzi** [comment]
@lasalvavida I should have thought of this sooner, but I created a [gltf-1.0.1](https://github.com/AnalyticalGraphicsInc/cesium/compare/gltf-1.0.1) branch which we can use to accumulate all glTF 1.0.1 updates, and then we'll merge to master in one go once the spec is official.

I also updated CHANGES.md in this branch so each PR does not need to provide fine-grained updates.

Can you please change this and all glTF PRs to target the gltf-1.0.1 branch?  This is a new option in the GitHub UI.

**Comment by lasalvavida** [comment]
Changed the base to the gltf 1.0.1 branch

**Comment by pjcozzi** [comment]
@lexaknyazev we are starting to implement glTF 1.0.1 in Cesium and gltf-pipeline.  This should help tighten up anything that we overlooked.  It's totally optional, but if you have the time, it would be great if you could look over these PRs to make sure we are keeping the spec/implementation in-sync.  We'll CC you on them; each PR is small.

File: Specs/Scene/ModelSpec.js [hunk 1]
```diff
@@ -1598,6 +1599,13 @@ defineSuite([
         });
     });
 
+    it('loads a glTF with WEB3D_quantized_attributes and accessor.normalized for higher precision decode matrix', function() {
```
Comment (pjcozzi) [inline Specs/Scene/ModelSpec.js]: I'm not following the "higher precision decode matrix."  Why not add per-vertex normalized `ubyte8` colors to an existing model since that is the common use case?  In general, I would also like to avoid bloating the repo with a lot of new test models to test corner cases if we are able to augment existing ones reasonably.

**Comment by lasalvavida** [comment]
> I'm not following the "higher precision decode matrix." Why not add per-vertex normalized ubyte8 colors to an existing model since that is the common use case? In general, I would also like to avoid bloating the repo with a lot of new test models to test corner cases if we are able to augment existing ones reasonably.

This has been updated to modify the existing quantized box model instead of adding a new one. Commits have been squashed so it is removed from the history. 

That may be a more common use case, but this also tests the functionality of `normalized` perfectly well, and now it doesn't add another model to the repo. 

See my [gltf-pipeline comment](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/pull/165#issuecomment-243499081) on why this is useful.

**Comment by lilleyse** [comment]
This looks good to me. I think @pjcozzi's suggestion is good since the testing can be more concrete, i.e. checking the pixel color vs. just checking it was rendered. It should be pretty easy to modify `Box-Color` to do the job, you may not even need to write new tests.

**Comment by lasalvavida** [comment]
Modified Box-Color to use the normalized property

**Comment by lilleyse** [comment]
Looks good! @pjcozzi do you want to do a quick final review?

**Comment by pjcozzi** [comment]
> See my gltf-pipeline comment on why this is useful.

I see, thanks!

**Comment by pjcozzi** [comment]
FYI - this does not update the `version` in Box-Color.gltf to `1.0.1`.  I think that is fine for now as we keep merging into `gltf-1.0.1`, then we'll update all the models.
